diff -Naur luksipc/luks.c luksipc_fixed/luks.c
--- luksipc/luks.c	2016-09-08 14:05:12.913817000 +0200
+++ luksipc_fixed/luks.c	2016-09-07 13:05:15.122000000 +0200
@@ -28,6 +28,7 @@
 #include <unistd.h>
 #include <errno.h>
 #include <inttypes.h>
+#include <libgen.h>
 
 #include "exec.h"
 #include "luks.h"
@@ -52,10 +53,13 @@
 /* Returns if the given device mapper name is available (i.e. not active at the
  * moment) */
 bool isLuksMapperAvailable(const char *aMapperName) {
+        char *bname;
+        bname = basename(aMapperName);
+
 	const char *arguments[] = {
 		"cryptsetup",
 		"status",
-		aMapperName,
+		bname,
 		NULL
 	};
 
